webrtc.html

```diff
@@ -614,16 +645,12 @@
          task ran -->
 
               <li>
-                <p>Remove <var>stream</var> from <var>connection</var>'s
-                <a href="#remote-streams-set">remote streams set</a>.</p>
+                <p>Remove the <code>RTCRtpReceiver</code> associated with <var>track</var> from
+                <var>connection</var>'s <a href="#receivers-set">set of receivers</a>.</p>
               </li>
 
               <li>
-                <p><a href="#fire-a-stream-event">Fire a stream event</a> named
-                <code title="event-MediaStream-removestream"><a href=
-                "#event-mediastream-removestream">removestream</a></code> with
-                <var title="">stream</var> at the <var>connection</var>
-                object.</p>
+                <p>End the <var>track</var> object.</p>
```

Comment:
The track was ended by the parent step 3 as well. The old step 4.3 fired the "removestream" event, which we have no counterpart for in the new approach, so I think it's fine to just remove step 4.3.